### DISCUSSION TITLE
Any plans to automatically recognize the new MP4 audible files?

### DISCUSSION THREAD

**Discussion by nurbles** [audiamus/AaxAudioConverter#115] [General]
I discovered on my own what I just found here:  Renaming an Audible download file from .MP4 to .AAX allows the converter to process it. Are there any plans to tweak the program to allow the MP4 extension to be used directly?  It would be fine if non-audible files are later rejected.  Renaming is OK, just kinda annoying.  (smile)

**Comment by audiamus** [comment]
Yes, I will allow .mp4 in the next version. I am using the same trick with [Book Lib Connect](https://github.com/audiamus/BookLibConnect) at the moment. Book Lib Connect decrypts to .m4b, which is just another alias for .mp4, suggesting that the content is audio only and not music (.m4a) but an audio book. AAX Audio Converter will then detect whether it needs to decrypt it or not. (If it has, it can only apply activation code, not the more sophisticated keys that come with .aaxc and for what we now have Book Lib Connect.) It also has to reject video .mp4.   

The plan is:

1. Fix any significant bugs in Book Lib Connect.
2. Create a last version of current AAX Audio Converter generation 1, that allows .mp4 input, fixes a few glitches and that also points out to Book Lib Connect.
3. Integrate Book Lib Connect into AAX Audio Converter and release it as AAX Audio Converter generation 2.
4. Add more features ...

**Reply by nurbles** [reply]
Looking forward to the next version, thanks!
